mw-webapp/src/component/sidebar/Sidebar.cy.tsx

```diff
@@ -0,0 +1,78 @@
+import {BrowserRouter} from "react-router-dom";
+import {Button} from "src/component/button/Button";
+import {NavigationLink, Sidebar} from "src/component/sidebar/Sidebar";
+import {getDataCy} from "src/utils/cyTesting/getDataCy";
+
+const SIDEBAR_CY = {
+  dataCyTrigger: "trigger",
+  dataCyContent: "content",
+};
+
+const SIDEBAR_TRIGGER = (
+  <Button
+    value="Sidebar trigger"
+    onClick={() => { }}
+  />);
+
+const SIDEBAR_LINKS: NavigationLink[] = [
+  {
+    path: "/",
+    value: "Home",
+  },
+  {
+    path: "/users",
+    value: "Users",
+  },
+];
+
+const SIDEBAR_CLOSE_BUTTON = "[class*=\"closeButton\"]";
```

Comment:
Why do we need this syntax, why but to use ".closeButton"

Does it possible to add data-cy attribute to close button

Reply:
Because we using  modules and it's part of className not full

Reply:
It's not possible right now

Reply:
Thanks, added